src/command_modules/azure-cli-monitor/azure/cli/command_modules/monitor/_params.py

```diff
@@ -112,21 +112,33 @@ def load_arguments(self, _):
         c.ignore('filter')
     # endregion
 
-    # region Diagnostics
```

Comment:
Unless you're going to remove all the # region/ # enregion pairs, you might as well keep this. It is handy when interpretted by Visual Studio.

Reply:
Added it back.